model/profile/registry.yaml

```diff
@@ -78,3 +78,36 @@ groups:
                 [Rust](https://wikipedia.org/wiki/Rust_(programming_language))
               value: "rust"
               stability: development
+  - id: registry.profile.pprof
+    type: attribute_group
```

Comment:
Does `attribute_group` type allow holding multiple values in a single attribute? With the `has_*` values a mapping can have several of them set.

Reply:
To my knowledge, a single attribute, like `profile.pprof.mapping`, can not have multiple values, like `has_filenames` and `has_functions`. There can be onlye one value per attribute. Consequently, we have to set an attribute per value.

I also asked in [#otel-semantic-conventions](https://cloud-native.slack.com/archives/C041APFBYQP/p1752821911097279) about this.

Reply:
To follow up on this: I'm not super familiar with the syntax of semantic conventions, but right now it feels like we define an attribute `profile.pprof.mapping` that can have one of `has_functions` / `has_filenames` / `has_line_numbers` / `has_inline_frames` values, while to be an equivalent to the pprof format's corresponding boolean fields, we need to be able to set each of these values independently.

Reply:
string array is also not possible if well-known values are defined.